app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db")
+        self.data_base = "cinema"
```

Comment:
The table name stored in `self.data_base` should be plural, such as 'cinemas', to match the checklist requirement that table names be plural.